class ContinuousFailureException(Exception):
    pass


class TimeoutException(Exception):
    pass
